range-touch.js

```diff
@@ -41,6 +41,10 @@
       e.type.toLowerCase() == 'mspointer'+type)
   }
 
+  function extractTouchesFromEvent(e) {
```

Comment:
Implemented this to return the touches list, whenever it's present on the event or in the original event.